data/account.go

```diff
@@ -65,3 +66,13 @@ type AccountESDT struct {
 	IsSender        bool
 	IsNFTOperation  bool
 }
+
+// ResponseBulk defines the structure for response accounts ESDT bulk
```

Comment:
`// ResponseBulk defines the generic structure for an Elasticsearch bulk request`
also, can be moved to a more generic file (not account.go) as we can use it in other places as well